data/transit/route/bus.json

```
@@ -3314,6 +3316,16 @@
       "locationSet": {"include": ["us"]},
       "note": "There are at least 4 different bus networks in US named 'DASH.' They should be split.",
       "tags": {"network": "DASH", "route": "bus"}
+    }
+    {
+      "displayName": "DASH",
+      "locationSet": {"include": ["us"]},
```

```suggestion
      "locationSet": {"include": ["lausd_los_angeles.geojson"]},
```
This will restrict this to [just LA county](https://github.com/osmlab/name-suggestion-index/blob/main/features/us/lausd_los_angeles.geojson) in order to prevent other DASHs from being matched with this as mentioned on line 3317